### DISCUSSION TITLE
Face restoration in replicate is not working.

### DISCUSSION THREAD

**Discussion by alexd725** [huggingface/diffusers#7402] [Q&A] [answered]
Hi guys.

Im trying to restore face using Replicate.
Im doing exact way from https://replicate.com/sczhou/codeformer?input=python
But not sure Why it doesn't work.

This is error condtent.
```
ReplicateError Details:\ntitle: Invalid version or not permitted\nstatus: 422\ndetail: The specified version does not exist (or perhaps you don't have permission to use it?)
```

Here is my code

```

REPLICATE_API_TOKEN = os.getenv("REPLICATE_API_TOKEN")
replicate = Client(REPLICATE_API_TOKEN)
output = replicate.run(
    "sczhou/codeformer:7de2ea26c616d5bf2245ad0d5e24f0ff9a6204578a5c876db53142edd9d2cd56",
    input={
        "image": "https://replicate.delivery/pbxt/IngEkQmZiZ3whtbkNAiOIdCsYAWVMkmoIBJnw7t2TPgvJn5S/photo.jpg",
        "upscale": 1,
        "face_upsample": True,
        "background_enhance": True,
        "codeformer_fidelity": 0.7
    }
)
print(output)
```
Pleaset help me with this issue. Thanks

**Comment by tolgacangoz** [comment, marked as answer]
I guess you need to pay for it. It says "Set up billing to run this model."
Other than that, I think it would be more appropriate to ask this question on Replicate's Discord channel for instance.

**Reply by alexd725** [reply]
Thanks

**Reply by tolgacangoz** [reply]
In case you missed it, there's also a Colab demo link on the same page.

**Reply by alexd725** [reply]
I got it. Thanks
